src/main.js

```diff
@@ -68,32 +66,37 @@ function createCardFromTemplate(userIdea) {
 
   userIdeas.appendChild(ideaCard)
   clearInputFields()
-  newIdea.saveToStorage()
+  updateLocalStorage()
 }
 
-function toggleStarred(event) {
-  // change card.star to true
+function toggleStarred() {
+  toggleStarProperty(event)
+  toggleStarIcon(event.target.parentElement.parentElement)
+  updateLocalStorage()
+}
 
+function toggleStarProperty(event) {
+  var iconAncestor = event.target.parentElement.parentElement
```

Comment:
I love this name! `iconAncestor` feels very very cool.

That said, this feels like _kind of_ a code smell. I can't tell why, exactly, just this kind of de-nesting could maybe use a different approach.